datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -23,11 +23,16 @@
 log = logging.getLogger('test_downloader')
 IntegrationMetadata = namedtuple("IntegrationMetadata", ["version", "root_layout_type"])
 
-# Integrations released for the last time by a revoked developper but not shipped anymore.
+# Integrations released for the last time by a revoked developer but not shipped anymore.
 EXCLUDED_INTEGRATIONS = [
     "datadog-docker-daemon",
+    "datadog-dd-cluster-agent",  # excluding this since actual integration is called `datadog-cluster-agent`
+    "datadog-kubernetes",  # excluding this since `kubernetes` check is Agent v5 only
 ]
 
+# Specific integration versions released for the last time by a revoked developer but not shipped anymore.
+EXCLUDED_INTEGRATION_VERSION = ["simple/datadog-ibm-mq/datadog_ibm_mq-4.1.0rc1-py2.py3-none-any.whl"]
```

Comment:
Hmm. Is this the latest version for the integration now?

Reply:
No, the actual latest version of IBM MQ integration is actually 3.20.0, I think when this wheel was made originally 2 years ago the dev probably meant to make a 1.4.0 wheel instead. 

Since this excluded version is 4.1.0-rc.1, we would need to release a v4.1.1 in order to not need this.

Reply:
Ok, so where is the download for this wheel being tested at all? Is it because if you don't specify a version number, the downloader [tries](https://github.com/DataDog/integrations-core/blob/61f1fb044db272f5c7bc030f202a919fa40b9e77/datadog_checks_downloader/datadog_checks/downloader/download.py#L319) to get the latest version, and this comes up accidentally?

Reply:
@sarah-witt That's a good idea, although there might be cases where the rc or beta wheel is the actual most recent wheel? I remember Agent 7.33 we had to release HTTP Check 6.1.2-rc.1 for it to backport some change.

@trishankatdatadog Yes I believe the downloader tries to get the latest version. The 4.1.0-rc.1 wheel is a mislabeled wheel made by a dev who's now revoked. Until we have 4.1.1,  the downloader will keep trying to test this wheel unless we exclude it.

Reply:
Yeah that edge case is true but a valid version would still be downloaded in that case. I'm fine with excluding these edge cases now and maybe discussing changing the behavior at a different time